### PR TITLE
Update/refactor rest of tests in integrations/user-management-api

### PR DESCRIPTION
**Pull request by josemigallas**
♻️ refactor a lot
✅ fixes deprecation params as positional arg
🚨 fixes lint (rubocop)
🎨 improves code format

### CODE REVIEW

**Review by nidhi-soni1104 — APPROVED** [review]
LGTM

File: test/integration/user-management-api/service_plan_features_test.rb [hunk 1]
```diff
@@ -75,20 +64,19 @@ def setup
   pending_test 'enable existing feature'
 
   test 'disable feature' do
```
Comment (akostadinov) [inline test/integration/user-management-api/service_plan_features_test.rb:66]: I can't spot the difference with `enable new feature` test. This also makes me wonder about naming of `disable non-existing feature` test. I remember discussing this with @josemigallas , maybe he had a PR about this as well?

**Review by akostadinov — COMMENTED** [review]
Looks great! Can we just clear up the dust with disable/enable feature?

File: test/integration/user-management-api/service_plan_features_test.rb (hunk 1 above)
Comment (josemigallas) [reply test/integration/user-management-api/service_plan_features_test.rb]: This file is such a mess 😅 I've updated it following API documentation. I think it makes much more sense now

**Review by hallelujah — APPROVED** [review]
That is a huge loads of file changes :) LGTM :rocket:

**Review by jsmadis — APPROVED** [review]

**Pull request merged**